gatsby-browser.js

```diff
@@ -1,4 +1,4 @@
 // custom typefaces
 import "typeface-montserrat";
 import "typeface-merriweather";
-require("prismjs/themes/prism-solarizedlight.css");
```

Comment:
if not maybe remove the dep?

Reply:
It’s a peerDep of `gatsby-remark-prismjs`